app/models/process_plate.rb

```diff
@@ -1,38 +1,28 @@
 # frozen_string_literal: true
-class ProcessPlate < ActiveRecord::Base
+class ProcessPlate < ApplicationRecord
   include ProcessPlateValidation
 
-  BARCODE_REGEX = /\S+/
+  BARCODE_REGEX = /\S+/.freeze
 
-  attr_accessor :api
-  attr_accessor :user_name
-  attr_accessor :witness_name
-  attr_accessor :instrument_used
-  attr_accessor :asset_search_results
-  # remove active record
```

Comment:
A load of unnecessary attr_accessors to handle the memoization pattern. Removed them